action/protocol/execution/evm/evm.go

```diff
@@ -399,7 +401,7 @@ func prepareStateDB(ctx context.Context, sm protocol.StateManager) (*StateDBAdap
 	)
 }
 
-func getChainConfig(g genesis.Blockchain, height uint64, id uint32, getBlockTime GetBlockTime) (*params.ChainConfig, error) {
+func getChainConfig(g genesis.Blockchain, height uint64, id uint32, getBlockTime func(uint64) (*time.Time, error)) (*params.ChainConfig, error) {
```

Comment:
change to `func(context.Context, uint64) (*time.Time, error)`? so L191 can simply use `blockHeightToTime()`